Composer/packages/lib/bot-deploy/src/botProjectDeploy.ts

```diff
@@ -89,6 +90,22 @@ export class BotProjectDeploy {
       value: scope,
     };
   }
+  private async getTenantId() {
+    if (!this.accessToken) {
+      throw new Error('Invalid Access Token');
```

Comment:
Should we say something more precise, such as

> Error: Missing access token. Please provide a non-expired Azure access token. Tokens can be obtained by running `az account get-access-token`

Doesn't have to be like that, but should be actionable for the user. The only reason the access token is null is because thye didn't provide it.